db/changes.go

```diff
@@ -774,9 +774,13 @@ func (db *Database) SimpleMultiChangesFeed(chans base.Set, options ChangesOption
 			}
 
 			if options.Revocations && db.user != nil {
-				channelsToRevoke := db.user.RevokedChannels(options.Since.SafeSequence())
-				for channel, triggeredBy := range channelsToRevoke {
-					feed := db.buildRevokedFeed(db.changeCache.getChannelCache().getSingleChannelCache(channel), options, triggeredBy, to)
+				revocationCheckSeq := options.Since.SafeSequence()
```

Comment:
Thinking about it more, I don't think we want to use SafeSequence() as revocationCheckSeq.  Consider the case:
- sequences 100, 110 and 120 are in channel foo
- user loses access to channel foo at sequence 115
- sequence 105 is skipped
A user that replicated when latestSequence is 112 would end up with a lastSeq value like `100::110` (lowSeq is 100, seq is 110).
If that user later issues a changes request with since=100::110 and detects the revocation, we'd want to revoke everything up to 110, not 100.

Reply:
I think that'd still be the case here no? So 100 the lowSeq would be used to say 'what channels did I have at 100' to calculate which channels to revoke. We still revoke everything up to 110

Reply:
I thought we only sent revocations post-since if the document was in the channel prior to the since, is that not the case?  And in this case, the doc with seq 110 wasn't necessarily in the channel at 100?

Reply:
We do but that since check uses the regular seq rather than this `revocationCheckSeq` value.

You may be right though, we might not want SafeSeq here.

Reply:
The post-since check is being done in buildRevokedFeed based on seq, so isn't affected by this - this should be fine as-is.